test/cliplugin/localkms/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements fake signer to be used in tests
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/sigstore/sigstore/pkg/signature/kms/cliplugin/handler"
+)
+
+const expectedProtocolVersion = "1"
+
+func main() {
+	// we log to stderr, not stdout. stdout is reserved for the plugin return value.
+	spew.Fdump(os.Stderr, os.Args)
```

Comment:
Yes, because most of the strict fields are pointers within nested structs. fmt will not resolve the values at pointers beyond the first level, but spew will.